# UMP07: Produktion von Bobby Cars

## Lösung LP in Matrix Notation
Nachfolgendes Skript zeigt die Lösung des Modells als LP, d.h. als lineares Programm, ausgehend von einer Formulierung in **Matrix-Notation**. 


In [2]:
from UMP07_LP_MATRIX_NOTATION import *

main()

Optimization terminated successfully. (HiGHS Status 7: Optimal)
Objective value:	-4360.0
Results:			[320. 360.]


**Achtung:** Für den Solver mussten wir die Zielfunktion auf eine Aufgabe vom Typ "minimiere..." umstellen. Daher muss der Zielfunktionswert mit (-1) multipliziert werden (Vorzeichenwechsel). Der Gewinn beträgt also 4360 CHF, wenn 320 Hasen (x1) und 360 Tiger (x2) produziert werden.